src/docs/integrations/msteams/index.mdx

```diff
@@ -9,60 +9,45 @@ You will need to be a Microsoft Teams Administrator to perform some of the requi
+| Long description          | enter anything              |
+| Developer or company name | Sentry                      |
+| Website                   | https://www.sentry.io       |
+| Privacy policy            | https://sentry.io/privacy/  |
+| Terms of use              | https://www.sentry.io/terms/|
 
-![Third Party Apps](./3rd-party-apps.png)
 
-## Create the Sentry App
+5. Hit "Save" at the bottom of the page.
+6. On the left hand sidebar click "App features" >> "Bot" >> "Create a new bot". You may name this whatever you like.
+7. Enter the bot endpoint address as `{YOUR_DOMAIN}/extensions/msteams/webhook/` and hit save. Take note of your bot ID (shown in the URL).
+8. Go to "Client secrets" and create a secret. Copy the secret as we'll use it later on.
+9. To go back click "Apps" >> {name of your app} >> "App features" >> "Bot" >> and then choose the bot you just created from the "Select an existing bot" dropdown and hit Save. 
+10. Click "Add a command" and call it anything you like. Select "Team" as the scope, Add it, and again hit Save. 
 
-Now you will need to create a Microsoft Teams app for the Sentry integration.
-
-1. Visit your [Microsoft Teams page](https://teams.microsoft.com/)
-2. Install the "App Studio" app from the Teams Store
-![App Studio](./app-studio.png)
-3. Go to App Studio
-4. Click on the "Manifest Editor" tab
-5. Press "Crete new app"
-![Create New App](./create-new-app.png)
-
-## Fill out Manifest
-1. Fill out the form on the "App details page" (put whatever you want, it doesn't impact functionality)
-2. Click on the "Bots" tab on the sidebar
-3. Press "Set up"
-4. Fill out the form as shown below
-![Set Up Bot](./set-up-bot.png)
-5. Press "Generate new password" 
-6. Store the password somewhere as it will be used as the `msteams.client-secret` later
-7. Take note of the ID of your bot as shown below as that will be used the `msteams.client-id` later
-![Client ID](./client-id.png)
-8. Set the "Bot endpoint address" to `{YOUR_DOMAIN}/extensions/msteams/webhook/`
 
 ## Set up config.yml
 
-Take the values from above and add them to the `config.yml` file as shown below
+Take the values from before and add them to the `config.yml` file as shown below:
 
 ```yml
 # Microsoft Teams #
-msteams.client-id: your-client-id
-msteams.client-secret: your-secret
+msteams.client-id: 'your-bot-client-id'
+msteams.client-secret: 'your-bot-secret'
 ```
 
 ## Installation
 
-If you left App Studio, go back and go to the "Manifest editor" tab and select your app
-
-1. Click on the "Test and distribute" tab on the left
-2. Fix any errors shown on the right side below "Description"
-![Errors](./errors.png)
-3. Press "Install"
-4. Press "Add to a team" in the popup that appears
-5. Pick the team(s) you want to install the app on
+1. At the top right of the screen click "Preview in Teams".
+2. Select "Add to a team" in the resulting popup, and choose the team(s) you want to install the app on.
```

Comment:
Okay after some troubleshooting what I had to do was click `Publish to org` in the sidebar
<img width="292" alt="image" src="https://user-images.githubusercontent.com/35509934/148294589-6569a7e3-e219-4934-96fb-a3e567a1af94.png">
Then headover to https://admin.teams.microsoft.com/policies/manage-apps, find the app in the list by sorting by Custom App 
<img width="1486" alt="image" src="https://user-images.githubusercontent.com/35509934/148294854-ee147c46-d696-4c2e-a810-0a76f6eacb9f.png">



and click its title
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/35509934/148294781-fbde3f04-1adc-46fd-87af-3037abe1cf14.png">
And publish it to the org-wide app catalog
Then I could find it under "Built for your org"
<img width="1486" alt="image" src="https://user-images.githubusercontent.com/35509934/148295075-a8b9276f-b21b-4b54-ba6c-bc35e624714a.png">

Reply:
Then once it was added to the team channel, I had to click Complete Setup:
![image](https://user-images.githubusercontent.com/35509934/148295355-69f04b00-bb48-43fd-8b3f-51aa7bf42d17.png)
Then select the organization I wanted to add it to:
<img width="873" alt="image" src="https://user-images.githubusercontent.com/35509934/148295195-1d63e147-15aa-4f4b-a4cd-b3bf85280a21.png">
And confirm it:
<img width="904" alt="image" src="https://user-images.githubusercontent.com/35509934/148295461-fcf0e75f-4e0e-47ec-acf5-ae09cba54500.png">

Reply:
I think some of these comments are outlined in the next paragraph, so it might be unnecessary to include them in the numbered list, but it's your call.